docs/astro/src/content/docs/reference/layouts/overview.mdx

```diff
@@ -3,3 +3,41 @@
 title: Common Properties
 description: Layout Common Properties
 ---
+
+import Link from '/src/components/Link.astro';
+import SlintProperty from '/src/components/SlintProperty.astro';
+
+
+## Properties
+These properties are valid on all visible items and can be used to specify constraints when used in layouts:
+
+### Col, Row
+<SlintProperty propName="col, row" typeName="int" default="0">
+See <Link type="GridLayout" />.
+</SlintProperty>
+
+### Colspan, Rowspan
```

Comment:
Minor detail I'm wondering about: Here it says "Colspan" and "Rowspan", but in the rest of the section it's camel case (MaxWidth, MinHeight, VerticalStretch).